Tests/WorkspaceTests/InitTests.swift

```diff
@@ -217,10 +217,10 @@ final class InitTests: XCTestCase {
             XCTAssertMatch(testFileContents, .contains(#"@Test func example() async throws"#))
             XCTAssertMatch(testFileContents, .contains("func testExample() throws"))
 
-            // Try building it
-            await XCTAssertBuilds(path)
-            let triple = try UserToolchain.default.targetTriple
-            XCTAssertFileExists(path.appending(components: ".build", triple.platformBuildPathComponent, "debug", "Modules", "Foo.swiftmodule"))
+            // Try building it -- DISABLED because we cannot pull the swift-testing repository from CI.
```

Comment:
Is there a reason `XCTSkipIfCI` is not used here instead?

Reply:
Please DM me.

Reply:
We don't want to skip the test. We just can't run all of it with an older toolchain.